### ISSUE TITLE
Warning: TypeError: binance.order is not a function

### ISSUE THREAD

**Issue by ManuCart** [jaggedsoft/node-binance-api#231]
**Title**
- help

**Short Description:**
- Warning: TypeError: binance.order is not a function
  
**Platform:**
- linux

**node version:**
- v10.4.0

**Long descrption**
- UnhandledPromiseRejectionWarning: TypeError: binance.order is not a function

**code**
```
orderId = binance.order(side, symbol, Q, price, {type:'LIMIT'}, (error, response) => {
  if (response="{}") {
  console.log(chalk.bgRed(symbol + " : " + side + "ERROR ORDER RETURN EMPTY DATA"))
  return;
}  else  {
  console.log("Limit "+side+" response", response);
  console.log("order id: " + response.orderId);
..
}
return response.orderId
})

```

thank you

**Comment by dmzoneill** [comment]
hey, they were some recent changes to constructor.

specifically 

const binance = new Binance();

how are you instantiating?

**Comment by ManuCart** [comment]
Thanks, but no way

When I use ````binance.sell```` or  ````binance.buy```` it's working.
But when I use ````binance.order```` I have the error

**Comment by jaggedsoft** [comment]
Is this a WAMP installation?

You probably meant to type `==` here: `if (response="{}") {`

**Comment by ManuCart** [comment]
It's not a WAMP installation.
Thanks it's mistake, but no way.

**Comment by ManuCart** [comment]
Code Working :
````
async function AOrder (side, symbol, price, Q, gap) {
if (side=="BUY"){Q=Quantity(symbol,price)}

binance.buy(symbol, Q, price, {type:'LIMIT'}, (error, response) => {
  if (response=="{}") {
  console.log(chalk.bgRed(symbol + " : " + side + "ERROR"))
  return;
}  else  {
  console.log("Limit "+side+" response", response);
  console.log("order id: " + response.orderId);
..
}
return response.orderId
})
`````
Code not Working
````
async function AOrder (side, symbol, price, Q, gap) {
if (side=="BUY"){Q=Quantity(symbol,price)}

binance.order(side,symbol, Q, price, {type:'LIMIT'}, (error, response) => {
  if (response=="{}") {
  console.log(chalk.bgRed(symbol + " : " + side + "ERROR"))
  return;
}  else  {
  console.log("Limit "+side+" response", response);
  console.log("order id: " + response.orderId);
..
}
return response.orderId
})
`````
Really Strange

**Comment by jaggedsoft** [comment]
Fixed https://github.com/jaggedsoft/node-binance-api/commit/d865907888f25a4016a9c2092e56071ec9ec0b7a
Available in v0.7.3

**Comment by ManuCart** [comment]
Thanks for all, you save my night
Love your code
